rest/endpoints/user_recordings/profile_recordings.yml

```diff
@@ -466,7 +466,68 @@ paths:
                     type: array
                     items:
                       $ref: "../../snippets/Epg/BroadcastSlotEntity.yml#/BroadcastSlotEntity"
-                      
+
+  # ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+  /user_profiles/{user_profile_id}/recorded_broadcast_slots/{broadcast_slot_id}:
```

Comment:
Can we move this one just below `/user_profiles/{user_profile_id}/recorded_broadcast_slots` endpoint, as they are a bit similar?